recommender/recommender.py

```diff
@@ -1044,6 +1047,22 @@ def parse_xml(cls, node, runtime, keys, _id_generator):  # pylint: disable=unuse
 
         return block
 
+    def load_recommender_from_lms(self):
+
+        recommendation_data = []
+
+        recommendation_getter = UserStateSummaryCache(self.course_id)
```

Comment:
This dependency is backwards. The recommender block should just be able to use `self.recommendations`, and then Studio should be piping that through the `UserStateSummaryCache` via the declared `FieldData`.

As written, this makes the recommender XBlock completely dependent on edx-platform.